docs/processing-status-openapi-spec.yaml

```diff
@@ -759,6 +759,162 @@ paths:
+      parameters:
+        - name: data_stream_id
+          in: query
+          required: true
+          schema:
+            type: string
+          description: Filter by the data stream identifier to retrieve the report counts for.
+        - name: data_stream_route
+          in: query
+          required: true
+          schema:
+            type: string
+          description: Filter by the data stream route that was provided with the upload.
+          example: routineImmunization
+        - name: date_start
+          in: query
+          schema:
+            type: string
+          description: >-
+            Start date of filter.  Format is YYYYMMDDTHHMMSSZ (UTC format,
+            inclusive).  Defaults to 3 months prior to today.
+          example: 20231020T14103000Z
+        - name: date_end
+          in: query
+          schema:
+            type: string
+          description: >-
+            End date of filter.  Format is YYYYMMDDTHHMMSSZ (UTC format,
+            exclusive).  Defaults to today.
+          example: 20231030T14103000Z
+      responses:
+        '200':
+          description: Report counts for the given criteria
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  counts:
+                    type: string
+                  query_time_millis:
+                    type: string
+        '400':
+          description: Bad request.  The destination identifier provided was not found.
+        '401':
+          description: Authorization information is missing or invalid.        
+  '/report/counts/hl7/directIndirectMessageCounts':
+    get:
+      tags:
+        - DEX Status
+      summary: Report counts for the given criteria.
+      description: >-
+        Provides report counts for the given criteria.
+      parameters:
+        - name: data_stream_id
+          in: query
+          required: true
+          schema:
+            type: string
+          description: Filter by the data stream identifier to retrieve the report counts for.
+        - name: data_stream_route
+          in: query
+          required: true
+          schema:
+            type: string
+          description: Filter by the data stream route that was provided with the upload.
+          example: routineImmunization
+        - name: date_start
+          in: query
+          schema:
+            type: string
+          description: >-
+            Start date of filter.  Format is YYYYMMDDTHHMMSSZ (UTC format,
+            inclusive).  Defaults to 3 months prior to today.
+          example: 20231020T14103000Z
+        - name: date_end
+          in: query
+          schema:
+            type: string
+          description: >-
+            End date of filter.  Format is YYYYMMDDTHHMMSSZ (UTC format,
+            exclusive).  Defaults to today.
+          example: 20231030T14103000Z
+      responses:
```

Comment:
Would you please add days_interval here as well?

Reply:
Added days_interval